src/components/selection.ts

```diff
@@ -176,6 +176,8 @@ export default class SelectionUtils {
       return rect;
     }
 
+    if (sel.rangeCount === 0) { return rect; }
```

Comment:
Following the code style guide: 

```suggestion
    if (sel.rangeCount === 0) { 
      return rect; 
    }
```